src/S0A_download_ASDF_MPI.py

```diff
@@ -77,229 +74,244 @@
 ##################################################
 # we expect no parameters need to be changed below
 
-# time tags
```

Comment:
No functional changes in this block, it all simply got put into the ```download()``` function below and hence every line was indented.